server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_4__add_logical_database_change_task.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS `logicaldatabase_database_change_execution_unit` (
```

Comment:
4.3.2.4 is occupied, 4.3.2.9 is available now

Reply:
okay, renamed it